src/ClientOptionsTrait.php

```diff
@@ -261,6 +261,7 @@ private static function determineMtlsEndpoint(string $apiEndpoint)
     private function createCredentialsWrapper($credentials, array $credentialsConfig, string $universeDomain)
     {
         if (is_null($credentials)) {
```

Comment:
I know this was written already but all the cases return on each `if`. What if we forego the `elseif` for all of them?:
``` php
if (is_null($credentials) {
    return CredentialsWrapper::buidl($credentialsConfig, universeDomain);
}

if (is_string($credentials) || is_array($credentials)) {
    return CredentialsWrapper::build(['keyFile' => $credentials] + $credentialsConfig, $universeDomain);
}

if ($credentials instanceof FetchAuthTokenInterface) {
    authHttpHandler = $credentialsConfig['authHttpHandler'] ?? null;
    return new CredentialsWrapper($credentials, $authHttpHandler, $universeDomain);
}

if ($credentials instanceof CredentialsWrapper) {
    return $credentials;
}

throw new ValidationException(
     'Unexpected value in $auth option, got: ' .
     print_r($credentials, true)
);
```

I find it easier to read and more pleasing.

Reply:
Good catch, I totally agree!